.travis.yml

```diff
@@ -10,5 +11,8 @@ env:
 script:
   - make test config=coverage defines=$DEFINES -j2
   - make test config=release defines=$DEFINES -j2
+  - make test cxxstd=c++98 -j2
+  - make test cxxstd=c++03 -j2
+  - make test cxxstd=c++11 -j2
 
```

Comment:
This is not a good way to do this. This redundantly runs the same builds with default defines, and I don't really think it's important to test all of these on Travis. Let's remove these for now? (with defaulting to c++11 below this should retain the current testing coverage)